### PR TITLE
Add patch dependency in container

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # whitelist
 branches:
   only:
-      - u/fjammes/DM-1146 
+      - master
+      - tickets/DM-5319
```
Comment (jhoblitt) [inline .travis.yml]: I assume this was for testing and may/should be removed?

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 
-sudo: required
+sudo: false
```
Comment (jhoblitt) [inline .travis.yml]: I think `sudo` technically should be `required`.  While it seems to work like this, the job output does show sudo being used.

**Comment by jhoblitt** [comment]
The travis gem warns about the `language` key not being present.

```
Warnings for .travis.yml:
[x] missing key language, defaulting to ruby
```

File: tests/debian/Dockerfile [hunk 3]
```diff
@@ -2,8 +2,8 @@ FROM qserv/stack:debian_latest
 MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
 USER root
-# MariaDB dependencies
-RUN apt-get install -y git cmake libncurses5-dev liblzma-dev zlib1g-dev
+# eups/eupspkg and MariaDB dependencies
+RUN apt-get update && apt-get install -y git patch cmake libncurses5-dev liblzma-dev zlib1g-dev
```
Comment (jhoblitt) [inline tests/debian/Dockerfile]: I often pass `-qq` to `apt-get` in scripts to cut down on the rather verbose default output.

File: .travis.yml (hunk 2 above)
Comment (fjammes) [inline .travis.yml]: Yes, doc recommends using `required`: [Travis/Docker doc](https://docs.travis-ci.com/user/docker/), but this prevents using caching...

**Pull request merged**